backend/dispatch.py

```diff
@@ -79,16 +79,6 @@ def handle_message(self, body, message):
                 },
             )
             res.apply_async()
-
-        if b['task'] == "save_UI_state":
```

Comment:
just wondering what this has been replaced with?

Reply:
It's being handled by the `save_teleoscope_state` function, although I plan on talking to Dhruv today about using class inheritance.